pkg/gofr/gofr.go

```diff
@@ -224,6 +237,33 @@ func (a *App) Run() {
 	wg.Wait()
 }
 
+func (a *App) pingGoFr(client *http.Client, s string) {
+	url := fmt.Sprint(gofrHost, shutServerPing)
+
+	if s == "start" {
+		url = fmt.Sprint(gofrHost, startServerPing)
+
+		a.container.Info("GoFr tracks running servers via telemetry. Set GOFR_TELEMETRY=false in configs to disable it.")
+	}
```

Comment:
the same code is duplicated to check the variable in current implementation 

I suggest either 
- having a `func (a *App) hasTelemetry() bool`
- always calling pingGofr, and check for GOFR_TELEMETRY then return